am/values.yaml

```diff
@@ -236,17 +236,37 @@ api:
 
   additionalPlugins:
 #    - https://path_to_plugin
-  ssl:
-    enabled: false
-  #  keystore:
-  #    type: jks # Supports jks, pem, pkcs12
-  #    path: ${gravitee.home}/security/keystore.jks
-  #    password: secret
-    clientAuth: false
-  #  truststore:
-  #    type: jks # Supports jks, pem, pkcs12
-  #    path: ${gravitee.home}/security/truststore.jks
-  #    password: secret
+  jetty:
+    secured: true
```

Comment:
Are you sure that the `secured` flag is `true` by default ?

Reply:
my bad. I assumed it was per default 🙈